pkgs/applications/misc/hubstaff/default.nix

```diff
@@ -1,26 +1,39 @@
 { stdenv, fetchurl, unzip, makeWrapper, libX11, zlib, libSM, libICE
 , libXext , freetype, libXrender, fontconfig, libXft, libXinerama
 , libXfixes, libXScrnSaver, libnotify, glib , gtk3, libappindicator-gtk3
-, curl }:
+, curl, fetchFromGitHub }:
 
 let
 
-  version = "1.3.1-ff75f26";
+  data = builtins.fromJSON (builtins.readFile ./revision.json);
+
+  inherit (data) version url sha256;
 
   rpath = stdenv.lib.makeLibraryPath
     [ libX11 zlib libSM libICE libXext freetype libXrender fontconfig libXft
       libXinerama stdenv.cc.cc.lib libnotify glib gtk3 libappindicator-gtk3
       curl libXfixes libXScrnSaver ];
 
+  LOCALE_ARCHIVE_2_26 =
+    let
+      # A random Nixpkgs revision *before* the default glibc
+      # was switched to version 2.27.x.
+      oldpkgsSrc = fetchFromGitHub {
+        owner = "nixos";
+        repo = "nixpkgs";
+        rev = "0252e6ca31c98182e841df494e6c9c4fb022c676";
+        sha256 = "1sr5a11sb26rgs1hmlwv5bxynw2pl5w4h5ic0qv3p2ppcpmxwykz";
+      };
+
+      oldpkgs = import oldpkgsSrc {};
+    in "${oldpkgs.glibc_2_26}/lib/locale/locale-archive";
+
```

Comment:
We can't import a different nixpkgs inside of nixpkgs, that is why the CI failed this build.